src/App.js

```diff
@@ -1,23 +1,122 @@
-import activities from './resources/schedule.json'
+import activities from "./resources/schedule.json";
 import dayjs from "dayjs";
+import { useEffect, useState } from "react";
 
 function App() {
+  const today = new Date();
+  var tomorrow = new Date(today + 1);
+  tomorrow.setDate(new Date().getDate() + 1);
```

Comment:
I don't understand what's going on here. Can you explain?

Reply:
I've fixed it. Ill commit the new changes